### PR TITLE
DataSourceVariable: Fix state property name to be more descriptive

### PR DESCRIPTION
**Pull request by torkelo**
Changes name of the query property to pluginId. The query name is only from the persisted state model, and the reason it's named query there is very old and becuse the data source variable model shared it's model definition with query variable model.

For this new object we should not be use this old name and use a more descriptive name.

like:

* pluginId

or

* dataSourceType

### CODE REVIEW

**Review by dprokop — APPROVED** [review]
Think this requires changelog update too, some people are already using this.

File: packages/scenes/src/variables/variants/DataSourceVariable.tsx [hunk 1]
```diff
@@ -12,7 +12,13 @@ import { VariableValueOption } from '../types';
 import { MultiValueVariable, MultiValueVariableState, VariableGetOptionsArgs } from './MultiValueVariable';
 
 export interface DataSourceVariableState extends MultiValueVariableState {
-  query: string;
+  /**
+   * Include all data source instance of this plugin type
```
[the remark this answers is not in the document]
Comment (torkelo) [reply packages/scenes/src/variables/variants/DataSourceVariable.tsx:16]: @dprokop yea, maybe it should read "Include all data source instances with this plugin id"

Comment (torkelo) [inline packages/scenes/src/variables/variants/DataSourceVariable.tsx:16]:
```suggestion
   * Include all data source instances with this plugin id
```

Comment (dprokop) [reply packages/scenes/src/variables/variants/DataSourceVariable.tsx]: Yes, better. Also - sorry I also misunderstood the change when I added this comment, thought it's about Query variable, and it was not :)

It makes sense to change this tho.

**Pull request merged**